tests/pact/bad/bad-import-unimported-reference.repl

```diff
@@ -0,0 +1,16 @@
+(begin-tx)
+
+(module f F
+  (defcap F () true)
+  (defun f () true)
+  (defun g () true)
+)
+
+(commit-tx)
+(begin-tx)
+
+(module g G
+  (use f [g])
+  (defcap G () true)
+  (defun g () (f))
+)
```

Comment:
In separate transactions, when `use`'ing a module, references to unimported module definitions is disallowed (note: this will succeed in the same tx by design)